src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -34,154 +37,223 @@ type State = AsyncComponent['state'] & {
+                <Actions>
+                  <StyledButton
+                    title={t('Edit Key')}
+                    label={t('Edit Key')}
+                    icon={<IconEdit />}
+                    onClick={this.handleOpenEditDialog(key)}
+                  />
+                  <StyledButton
+                    title={t('Delete Key')}
+                    label={t('Delete Key')}
+                    onClick={this.handleDelete(key)}
                     icon={<IconDelete />}
                   />
-                </Content>
-              )
-            )}
-          </PanelBody>
-          <PanelAction>
-            <Button href={RELAY_DOCS_LINK} target="_blank">
-              {t('Read the docs')}
-            </Button>
-            <Button onClick={this.handleAdd} priority="primary">
-              {t('Add Relay')}
-            </Button>
-          </PanelAction>
-        </Panel>
+                </Actions>
+              </React.Fragment>
+            );
+          })}
+        </StyledPanelTable>
       </React.Fragment>
     );
   }
 }
 
 export default Relays;
 
-const Content = styled('div')`
-  display: grid;
-  grid-template-columns: 1fr max-content;
-  align-items: center;
-  border-bottom: 1px solid ${p => p.theme.borderDark};
-  padding: ${space(1)} ${space(2)};
-  :last-child {
-    border-bottom: 0;
+const StyledPanelTable = styled(PanelTable)`
+  grid-template-columns: repeat(3, auto) max-content;
+  > * {
+    padding: ${space(1)};
+    @media (min-width: ${theme.breakpoints[0]}) {
+      padding: ${space(1)} ${space(2)};
+    }
   }
 `;
 
-const Info = styled('div')`
+const KeyWrapper = styled('div')`
   display: grid;
-  grid-template-columns: repeat(4, 1fr);
+  grid-template-columns: auto 1fr;
   grid-gap: ${space(1)};
-  > *:nth-child(1),
-  > *:nth-child(2) {
-    grid-column: span 4;
-  }
+  align-items: center;
 `;
 
-const InfoItem = styled('div')`
+const IconWrapper = styled('div')`
+  justify-content: flex-start;
   display: flex;
-  align-items: center;
-  height: 100%;
+  cursor: pointer;
 `;
 
-const PanelAction = styled('div')`
-  padding: ${space(1)} ${space(2)};
+const Text = styled(TextOverflow)`
+  color: ${p => p.theme.gray700};
+  line-height: 40px;
+`;
+
+const Key = styled(Text)<{content: string}>`
+  visibility: hidden;
+  position: relative;
+  :after {
+    position: absolute;
+    top: 4px;
+    left: 0;
+    content: '${p => p.content}';
+    visibility: visible;
+    ${overflowEllipsis};
+  }
+`;
+
+const Actions = styled('div')`
   display: grid;
+  grid-template-columns: auto 1fr;
   grid-gap: ${space(1)};
-  grid-template-columns: auto auto;
-  justify-content: flex-end;
-  border-top: 1px solid ${p => p.theme.borderDark};
+  align-items: center;
 `;
 
-const Name = styled('h4')`
-  font-size: ${p => p.theme.fontSizeLarge} !important;
-  font-weight: 600;
-  margin-bottom: 0 !important;
-  color: ${p => p.theme.gray600};
+const StyledButton = styled(Button)`
+  width: 40px;
```

Comment:
I would really love it if we could do these action buttons without additional styles. 
It looks like we are moving to this kind of design in settings more and more, so it would be great not having to override them on every page.

Do you know if passing size="small" to a button and size="sm" to an icon would be close enough to design?
I did it that way in Source Maps settings. @doralchan wdyt?

Reply:
sure..let me update it :)